content/static/stylesheets/default.sass

```diff
@@ -130,6 +130,23 @@ img
       .ui.header a
         color: inherit
 
+/* Position, width, margin modifications for right rail
+/* Only for screen wider than 1200px
```

Comment:
I'm not quite sure whether I asked the right question. You write "Only for screen wider than 1200px". But if the code only works for "screen wider than 1200px", should we wrap it in a CSS media query?

Ref:
https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries